### DISCUSSION TITLE
Vyper Tutorials?

### DISCUSSION THREAD

**Discussion by howardpen9** [vyperlang/vyper#2247] [Ideas]
Can we have more Tutorial Example for Smart Contract using Vyper?

I'm new at compile Smart Contract!

**Comment by fubuloubu** [comment]
@howardpen9 please try ~~https://vyper.fun~~ https://learn.vyperlang.org if you are looking for a tutorial

**Reply by MetaB0y** [reply]
It's currently down I think?

**Reply by charles-cooper** [reply]
Now hosted at learn.vyperlang.org

**Reply by josealonso** [reply]
> Now hosted at learn.vyperlang.org

That server is down.

**Reply by charles-cooper** [reply]
> That server is down.


can you post a screenshot of what you are seeing? it loads ok for me.

**Comment by Stefs-2142** [comment]
> @howardpen9 please try https://vyper.fun if you are looking for a tutorial

Site can't be reached ;(

**Reply by fubuloubu** [reply]
Try https://learn.vyperlang.org/#/ instead

**Comment by fubuloubu** [comment]
Also, there's vyper content available on https://academy.apeworx.io
